index.html

```diff
@@ -49,6 +50,140 @@ <h2 class="connect"> <a href="#">Let's connect</a></h2>
           </ul>
         </div>
       </section>
+    </div>
+
+
+    <section class="works">
+      <article class="card card-1">
```

Comment:
- [ ] Kindly make the image and paragraph responsive for mobile devices.
![Captura de pantalla_2021-09-22_17-19-38](https://user-images.githubusercontent.com/53111464/134430626-7e05a266-bee3-4334-badb-cef595683d48.png)